### PR TITLE
Rohit/fixing token types

### PR DESCRIPTION
**Pull request by ChicK00o**

### CODE REVIEW

File: src/canvas.js [hunk 1]
```diff
@@ -223,8 +223,8 @@ class Canvas {
             }
             try {
               await this.refreshUserToken(this.refreshToken);
-            } catch (err) {
-              console.error(err);
+            } catch (err1) {
```
Comment (saurabhatquizizz) [inline src/canvas.js:226]: why this change ?

File: src/common.d.ts [hunk 2]
```diff
@@ -1,11 +1,15 @@
...
+  expires_in?: Number, 
+  refresh_token?: string,
+  lastRefresh?: Date,
+  info?: any,
 }
 
-export type GetUserToken = (userId: string) => Promise<Tokens>;
-export type SetUserToken = (userId: string, tokens: Tokens) => Promise<void>;
+export type GetAccessToken = (userId: string) => Promise<Tokens>;
+export type SetAccessToken = (userId: string, token: Tokens) => Promise<any>;
```
Comment (saurabhatquizizz) [inline src/common.d.ts:12]: what does setAccessToken actually return ? earlier it was void that why asking, are we returning anything now ?

File: src/canvas.js (hunk 1 above)
Comment (ChicK00o) [reply src/canvas.js:226]: Its one of the errors in eslint, its overriding the err, that is declared in the above scope, and was causing this lint issue, so just changed it.

File: src/common.d.ts (hunk 2 above)
Comment (ChicK00o) [reply src/common.d.ts:12]: yes, i checked the code, with setToken, we return the whole of the user object, as per the current contract, void was wrong previously. This is just type definition, and not implementation change, hence did not break.

**Review by saurabhatquizizz — APPROVED** [review]
looks good
